packages/material/scss/treeview/_variables.scss

```diff
@@ -6,15 +6,15 @@ $kendo-treeview-font-family: $font-family !default;
 /// Font size of the treeview component.
 /// @group treeview
 $kendo-treeview-font-size: $font-size-lg !default;
-$kendo-treeview-font-size-sm: $font-size-lg !default;
-$kendo-treeview-font-size-md: $font-size-lg !default;
+$kendo-treeview-font-size-sm: $font-size-md !default;
+$kendo-treeview-font-size-md: $font-size-md !default;
 $kendo-treeview-font-size-lg: $font-size-lg !default;
 /// Line height of the treeview component.
 /// @group treeview
 $kendo-treeview-line-height: 1.25 !default;
-$kendo-treeview-line-height-sm: 1.25 !default;
-$kendo-treeview-line-height-md: 1.25 !default;
-$kendo-treeview-line-height-lg: 1.25 !default;
+$kendo-treeview-line-height-sm: ( 20 / 14 ) !default;
+$kendo-treeview-line-height-md: ( 20 / 14 ) !default;
+$kendo-treeview-line-height-lg: 1.5 !default;
```

Comment:
we can use [$line-height-lg](https://github.com/telerik/kendo-themes/blob/develop/packages/material/scss/typography/_variables.scss#L30) here as the values are matching.